README.md

```diff
@@ -2,23 +2,32 @@
 
 This repository is designed to be a learning tool for learning the usage and best practices of the Python unittest package 
 
+## Prerequisites
+
++ [Python 3.x](https://docs.python.org/3/using/)
++ [Coverage](https://coverage.readthedocs.io/en/coverage-5.3/install.html)
+
 
```

Comment:
Looks good, I like the prereqs as they should give everyone what they need to get started.